Lib/test/test_logging.py

```diff
@@ -4853,6 +4887,30 @@ def dummy_handle_error(record):
             # didn't write anything due to the encoding error
             self.assertEqual(data, r'')
 
+    def test_log_taskName(self):
+        async def log_record():
+            logging.warning('hello world')
+
+        try:
+            encoding = 'utf-8'
+            logging.basicConfig(filename='test.log', errors='strict', encoding=encoding,
+                                format='%(taskName)s - %(message)s', level=logging.WARNING)
+
+            self.assertEqual(len(logging.root.handlers), 1)
+            handler = logging.root.handlers[0]
+            self.assertIsInstance(handler, logging.FileHandler)
+
+            with asyncio.Runner(debug=True) as runner:
+                logging.logAsyncioTasks = True
+                runner.run(log_record())
+        finally:
+            asyncio.set_event_loop_policy(None)
+            handler.close()
+            with open('test.log', encoding='utf-8') as f:
+                data = f.read().strip()
+            os.remove('test.log')
+            self.assertEqual('Task-1 - hello world', data)
```

Comment:
How can we be sure the task name will always be `Task-1`? What if more asyncio tasks are added in other tests which run before this one? Will the numbering still start at 1?

Reply:
Have changed it to an `assertRegex`, similar to:
https://github.com/python/cpython/blob/1f134e96ba994bea3aaea533d4a558df3c5cc3c0/Lib/test/test_asyncio/test_tasks.py#L393-L394